include/core/SFMLHeaders.hpp

```diff
@@ -0,0 +1,7 @@
+#ifndef SFMLHEADERS_HPP_
+#define SFMLHEADERS_HPP_
+
+#include <SFML/Graphics.hpp>
+
+#endif // SFMLHEADERS_HPP_
+
```

Comment:
This header is useless since unlike SDL, SFML includes are portable.

Reply:
Changed to use only necessary SFML includes directly